public/app/js/feedback.js

```diff
@@ -0,0 +1,32 @@
+$('.slick').slick({
```

Comment:
Để vào trong function.ready nhé
```suggestion
$('.slick').slick({
```